src/server/src/modules/job.ts

```diff
@@ -132,6 +135,13 @@ export const getJobs = (
         let year = modStart.getFullYear();
         modStart = month + '/' + date + '/' + year;
     }
+
+    const jobApplications = await getJobApplications(studentId);
+
+    if (!jobApplications) return undefined;
```

Comment:
Why you return undefined if no job applications from the student are found?

Reply:
If no job applications are found, the jobApplications array is empty without being undefined. The array is undefined only when the studentId passed in as an argument does not exist, which should never happen in this context.

Reply:
ohhhhhhh, can you just add a comment that explains that? I think it'd be good to clear it up.

Reply:
After that feel free merge the pr :D